### DISCUSSION TITLE
Ignore folders containing a specific file

### DISCUSSION THREAD

**Discussion by moeenio** [borgbase/vorta#2059] [FAQ] [answered]
Hello, I can no longer find the option to ignore folders based on the existence of a file with a defined name in Vorta 0.9.1. I used this option when I previously used Vorta 0.8.12. Has it been removed and why?

**Comment by shivansh02** [comment, marked as answer]
Hi! Exclude-if-present was indeed removed but has been added back. It should be a part of the next release.

**Reply by Rishi15** [reply]
Hi Shivansh, Can you suggest how do I use it in vorta?
